vuln/npm/new.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": 484,
+  "title": "webpack-dev-server origin validation",
```

Comment:
this needs to be set to the [CWE](https://cwe.mitre.org/), you can see previous vulns as references

Reply:
sounds good.  The best matching CWE I think is [CWE-346 Origin Validation Error](https://cwe.mitre.org/data/definitions/346.html).  Open to other interpretations the CWE system leaves some room for interpretation :)